examples/package-lock.json

```diff
@@ -0,0 +1,5646 @@
+{
```

Comment:
AFAIR we don't want to have so big package-lock.json files in our repo, do we?

Reply:
Please see readme of this PR.

Reply:
Oh, silly me